x/profiles/client/rest/tx.go

```diff
@@ -14,8 +14,10 @@ import (
 func registerTxRoutes(cliCtx context.CLIContext, r *mux.Router) {
 	r.HandleFunc("/profiles/{address}", saveProfileHandler(cliCtx)).Methods("PUT")
 	r.HandleFunc("/profiles/{address}", deleteProfileHandler(cliCtx)).Methods("DELETE")
-	r.HandleFunc("/dtag-transfer-requests", requestDTagTransferHandler(cliCtx)).Methods("POST")
-	r.HandleFunc("/dtag-transfer-requests/accept", acceptTransferRequestHandler(cliCtx)).Methods("POST")
+	r.HandleFunc("/profiles/dtag/request/{receiverAddress}", requestDTagTransferHandler(cliCtx)).Methods("POST")
+	r.HandleFunc("/profiles/dtag/accept/{senderAddress}", acceptTransferRequestHandler(cliCtx)).Methods("POST")
+	r.HandleFunc("/profiles/dtag/refuse/{senderAddress}", refuseDTagTransferRequestHandler(cliCtx)).Methods("POST")
+	r.HandleFunc("/profiles/dtag/cancel/{receiverAddress}", cancelDTagTransferRequestHandler(cliCtx)).Methods("POST")
```

Comment:
I think that these can be further improved by writing them as

 ```suggestion
	r.HandleFunc("/profiles/dtagrequests", requestDTagTransferHandler(cliCtx)).Methods("POST")
	r.HandleFunc("/profiles/dtagrequests/{senderAddress}/acceptances", acceptTransferRequestHandler(cliCtx)).Methods("POST")
	r.HandleFunc("/profiles/dtagrequests/{senderAddress}", refuseDTagTransferRequestHandler(cliCtx)).Methods("DELETE")
	r.HandleFunc("/profiles/dtagrequests/{receiverAddress}", cancelDTagTransferRequestHandler(cliCtx)).Methods("DELETE")
```